<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicTacToeWindow</class>
 <widget class="QMainWindow" name="TicTacToeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TicTacToeWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Interacive Button TicTacToe</string>
    </property>
    <widget class="QPushButton" name="board00">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board01">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board02">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board12">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board10">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board22">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board20">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board21">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="IBStatusLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Text-Based TicTacToe</string>
    </property>
    <widget class="QLabel" name="boardLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>181</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;---&lt;br/&gt;---&lt;br/&gt;---&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="TBStatusLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="rowInput">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="colInput">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>190</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Row (0-2)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>220</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Col (0-2)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Put</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
